### PR TITLE
feat: async value props should propagate when editor init finished

### PR DESCRIPTION
**Pull request by lwenn**
It fixes an unstable bug: sometimes the editor cannot render value props that fetch from service.
The reason of this bug is:
* when tinymce `setup()` is invoked, `initialized` of editor is `undefined`
* the order of `setup()` and `componentDidUpdate()` is unsure
* this make `componentDidUpdate()` break, because of `initialized` is `undefined`, then content cannot update

Thus, I add a `init_instance_callback` hook to update value from props.

### CODE REVIEW

File: package.json
Comment (tiny-james) [inline package.json:58]: Please remove the dependency on chromedriver and rollback the changes to the yarn.lock.

File: src/main/ts/components/Editor.tsx
Comment (tiny-james) [inline src/main/ts/components/Editor.tsx:219]: This method is good but you need to check if the user defined `init_instance_callback` themselves and call it if they did.

**Comment by tiny-james** [comment]
Thanks for the PR. 

To get it merged please rollback the changes to the yarn.lock and the package.json. I also noted one regression but I can fix that after merging if you want.

Please sign the CLA, I can't merge it at all without it.

File: package.json
Comment (tiny-james) [reply package.json:58]: Generally we won't accept PRs with any changes to the yarn.lock as it's too hard to review and can be used to sneak in bad dependencies.

**Comment by lwenn** [comment]
> Thanks for the PR.
> 
> To get it merged please rollback the changes to the yarn.lock and the package.json. I also noted one regression but I can fix that after merging if you want.
> 
> Please sign the CLA, I can't merge it at all without it.

It's all done, please review again.

**Review by tiny-james — APPROVED** [review]

**Review by techtangents — APPROVED** [review]

**Pull request merged**
